Sources/TuistDependencies/SwiftPackageManager/Models/PackageInfo.swift

```diff
@@ -148,7 +148,7 @@ extension PackageInfo {
         let resources: [Resource]
 
         /// The exclude patterns.
-        let exclude: [String]
+        let exclude: [String] // TODO: tests
```

Comment:
Tests should be added as part of the PR before merging them. If we encourage leaving TODO comments we might end up with many half-done features across the codebase.